crates/dbsp/src/operator/communication/exchange.rs

```diff
@@ -1001,6 +1014,20 @@ where
     fn fixedpoint(&self, _scope: Scope) -> bool {
         true
     }
+
+    fn flush(&mut self) {
+        // println!("{} exchange_receiver::flush", Runtime::worker_index());
+        self.flush_complete = false;
```

Comment:
this is rather unintuitive.
maybe the function should be called start_flush?

Reply:
At the circuit level it is now called `start_commit_transaction`. It ends up invoking `flush` on each operator in the circuit in some order. I am debating renaming this operator method to `start_commit_transaction` as well. `start_flush` is ok too.